desci-server/src/types/documents.ts

```diff
@@ -3,4 +3,5 @@ import { ResearchObjectV1 } from '@desci-labs/desci-models';
 export interface ResearchObjectDocument {
   manifest: ResearchObjectV1;
   uuid: string;
+  driveClock?: string;
```

Comment:
the driveClock field should be set in the initial document creation change  here in `getNodeDocument` and `createManifestDocument`: 

here:
- https://github.com/desci-labs/nodes/blob/develop/desci-server/src/services/manifestRepo.ts#L34
- https://github.com/desci-labs/nodes/blob/develop/desci-server/src/controllers/nodes/documents.ts#L41

<img width="925" alt="Screenshot 2024-01-15 at 12 13 39" src="https://github.com/desci-labs/nodes/assets/31709531/f844e1f1-0bf0-4ccd-9b91-43952714c4ff">